samples/boards/stm32/power_mgmt/blinky_standby_shutdown_mode/src/main.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/hwinfo.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/pm/pm.h>
+
+
+#define SLEEP_TIME_MS   300
+
+
```

Comment:
#if !defined(CONFIG_SOC_SERIES_STM32L4X)
#error Not implemented for other series
#endif